### ISSUE TITLE
Mobile: sidebar doesn't minimise when navigating to a new route

### ISSUE THREAD

**Issue by calumbell** [open5e/open5e#551]
## Observed Behaviour
(including reproduction)

The issue can be observed while using the sidebar navigation on mobile or at narrow screen widths on the live site. When you click on a link in on a link the navbar doesn't. This is creates quite an awkward user-experience as the user will need to click for a second time (away the navbar) actually view a page.

![ScreenRecording2024-08-06at12 14 00-ezgif com-optimize](https://github.com/user-attachments/assets/b4ddebd0-3076-4c3c-a09a-5c128bbc38ce)

## Desired Behaviour

At narrow screen widths, the navbar should minimise whenever the user navigates to a new page

**Comment by okrayum** [comment]
Hello, I'd like to tackle this issue if it is available. It looks like just a simple fix to the existing watcher to close the sidebar.

**Comment by calumbell** [comment]
Don't think anybody is currently working on this issue, I'll go ahead and get it assigned to you. Good luck! Give us an @ if you need any help

**Comment by okrayum** [comment]
Thank you @calumbell . I have a fix ready but am encountering the following error when making the commit:

![Screenshot 2024-08-07 115114](https://github.com/user-attachments/assets/a1196b75-c0d1-42c0-b1db-fb8f5de1c5c0)

The changes I made are in the layouts/default.vue file. Could you provide guidance on how to resolve this?
Thanks!
@okrayum

**Comment by calumbell** [comment]
Hey @okrayum, looks like this is an issue with your terminal is having trouble parsing the `.` between `.gitignore` and `--fix` in the shell command:

`eslint --ext '.js,.vue' --ignore-path .gitignore . --fix`

I am looking at the direction of the slashes in your screenshot and I am going to that you are using a Powershell terminal? Which I believe handles paths in a slightly different way to the Unix/Bash terminals I am more used to working in.

I did a little digging around online, and one potential solution would be to update the linting commands themselves for that they run on both Unix and Powershell. I believe this can be done by adding a slash after the period in the following lines in the `package.json` file:

**CURRENT**

```
{
  ...
  "scripts": {
    ...
    "lint:ci": "eslint --ext .js,.vue --ignore-path .gitignore .",
    "lint:js": "eslint --ext '.js,.vue' --ignore-path .gitignore .",
    ...
}
```

**NEW**

```
{
  ...
  "scripts": {
    ...
    "lint:ci": "eslint --ext .js,.vue --ignore-path .gitignore ./",
    "lint:js": "eslint --ext '.js,.vue' --ignore-path .gitignore ./",
    ...
}
```

Testing this out on local, adding the extra slashes doesn't appear to change how any of the pre-commit checks run on Unix/Bash. If you find that these changes work for you, go ahead and add the changes to `package.json` to this commit.

If it doesn't work, perhaps try commiting using a Unix terminal?

Good luck!

**Comment by okrayum** [comment]
Thank you for pointing me in the right direction. I was able to make the commit. I had to use double quotes and escape them in place of the single quotes here:
(I just made those changes locally and didn't push)

```json
"lint:js": "eslint --ext \".js,.vue\" --ignore-path .gitignore .",
"lint:prettier": "prettier --check --ignore-path .gitignore '**/*.{js,vue}'",

**Comment by calumbell** [comment]
This issue was resolved in PR #552

**Comment by okrayum** [comment]
@calumbell, Thanks! I'm glad I could help.
